h/services/links.py

```diff
@@ -19,7 +59,6 @@ def __init__(self, base_url, registry):
         :type registry: pyramid.registry.Registry
         """
         self.base_url = base_url
-        self.registry = registry
 
         # It would be absolutely fair if at this point you asked yourself any
```

Comment:
This request object and the comment below is the only reason to keep this a service, not just functions.

I don't want to go on a huge refactoring path, IMO keeping the service and the functions in the same module is a bit clearer.